src/crimson/os/seastore/segment_manager/zns.h

```diff
@@ -63,6 +63,13 @@ namespace crimson::os::seastore::segment_manager::zns {
   using write_ertr = crimson::errorator<crimson::ct_error::input_output_error>;
   using read_ertr = crimson::errorator<crimson::ct_error::input_output_error>;
 
+  enum class zone_op_t {
```

Comment:
nit, s/zone_op_t/zone_op/.

Reply:
Ok, I saw that in most of the places enums was defined like this(abc_t), so I followed the existing conventions.
I will change it.

Reply:
yeah, i believe the `_t` postfix dated back to the old days when we still need to `typedef` an enum for using it with the typedef'ed name. but since we are using C++, there is no need to typedef an enum anymore.

Reply:
Ok, updated with the new changes.